### PR TITLE
Contributors: stabilizing component

### PR DESCRIPTION
**Pull request by Garneauma**

### CODE REVIEW

**Comment by Garneauma** [comment]
Pre-approved upon successful review.

**Review by duboisp — CHANGES_REQUESTED** [review]
See the inline comments.


**Question for DTO:** Should we limit the width when there is long name for the institution that are take the full page width? Especially in medium view and up? Currently if the item takes more place it is moved on a new line, is that the expected behaviour?
image

File: components/gc-contributors/_base.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* Contributors */
+
+.gc-contributors {
+	margin-top: 38px;
+
+	h2, ul {
+		font-size: $small-size;
```
[the remark this answers is not in the document]
Comment (Garneauma) [reply components/gc-contributors/_base.scss:7]: This is specific to this component, not to the Topic template.

**Comment by Garneauma** [comment]
> See the inline comments.
> 
> **Question for DTO:** Should we limit the width when there is long name for the institution that are take the full page width? Especially in medium view and up? Currently if the item takes more place it is moved on a new line, is that the expected behaviour?

Asked DTO. The bullet should be on the right of the first item. Also, the second row should be left-aligned with the first row (not the "From:" title). Made the changes to reflect those decisions.

**Review by duboisp — CHANGES_REQUESTED** [review]
Can we bind the bullet with the following item instead of the preceding?

Because with a long department/agency name I do observe the following:

image

And we are missing a working example for that. May be a new sample code that you can add in the same working example page?

**Comment by Garneauma** [comment]
> Can we bind the bullet with the following item instead of the preceding?
> 
> Because with a long department/agency name I do observe the following:

The current behaviour for wrapping contributors names is the one decided by DTO.

**Review by duboisp — APPROVED** [review]
Look good, requested change has been applied

**Pull request merged**